src/alpaka_job_coverage/validate.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python3
+
+import argparse
+from packaging import version as pk_version
+from typing import Dict, Tuple, Union
+from typeguard import typechecked
+from io import StringIO
+
+from alpaka_job_coverage.globals import *  # pylint: disable=wildcard-import,unused-wildcard-import
+import alpaka_job_coverage.filter_compiler_name as ajc_compiler_name
+import alpaka_job_coverage.filter_compiler_version as ajc_compiler_version
+import alpaka_job_coverage.filter_backend_version as ajc_backend_version
+import alpaka_job_coverage.filter_software_dependency as ajc_software_dependency
+from alpaka_job_coverage.versions import is_supported_version
+
+
+@typechecked
+def cs(text: str, color: str):
+    """Print the text in a different color on the command line. The text after
+       the function has the default color of the command line.
+
+    :param text: text to be colored
+    :type text: str
+    :param color: Name of the color. If wrong color or empty, use default color
+                  of the command line.
+    :type color: str
+    :returns:
+    :rtype:
+
+    """
+    if color is None:
+        return text
+
+    output = ""
+    if color == "Red":
+        output += "\033[0;31m"
+    elif color == "Green":
+        output += "\033[0;32m"
+    elif color == "Yellow":
+        output += "\033[1;33m"
+    else:
+        return text
+
+    return output + text + "\033[0m"
+
+
+@typechecked
+def exit_error(text: str):
+    """Print error message and exit application with error code 1.
```

Comment:
```suggestion
    """Prints error message and exits application with error code 1.
```

Is there no `EXIT_FAILURE` in Python?

Reply:
There is something similar: https://docs.python.org/3.4/library/os.html#os.EX_OK
But it is UNIX only